src/Mod/Arch/ArchWall.py

```diff
@@ -1342,10 +1342,10 @@ def getExtrusionData(self,obj):
                             if curAligns == "Left":
 
                                 if layers:
-                                    curWidth = abs(layers[i])
+                                    curWidth = [abs(layers[i])]
                                     off = off+layeroffset
-                                    dvec.multiply(curWidth)
-                                    layeroffset += abs(curWidth)
+                                    dvec.multiply(curWidth[0])
+                                    layeroffset += abs(curWidth[0])
```

Comment:
Actually, this is a bad fix. There should be `widths`:

```
widths = []
...
curWidth = abs(layers[i])
...
widths.append(curWidth)
```

Reply:
@yorikvanhavre This fixes the crash, but maybe the intention was something else. Didn't find any tests that would guide on the intended behavior.